requirements.txt

```diff
@@ -1,3 +1,4 @@
+git+https://github.com/ONSdigital/ras-common.git@US032b/case-service-logging
```

Comment:
This is part of the integration piece and is required for the time being as the feature branch being referenced is still in the PR column. I will update this once the PR has been merged to master.